### PR TITLE
Updated for Michael Jenkins

### PR DESCRIPTION
**Pull request by michaeljenkins88**
added the Union Jack!

### CODE REVIEW

**Review by jravellana — COMMENTED** [review]
Nice emoji choice!

File: 2020_Fall/readme.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Huang, Valerie
 
-- [ ] Jenkins, Michael
+- [x ] Jenkins, Michael 🇬🇧
```
Comment (joewadcan) [inline 2020_Fall/readme.md:62]: I think you've got an extra space there after the `x`. Just make another edit (which will create another commit on this branch).

**Comment by joewadcan** [comment]
Hi @michaeljenkins88 - I think @stalerat merged this PR a bit too early. Can you submit another pull request with your GitHub username (and correct the extra space as well).

**Comment by joewadcan** [comment]
(nevermind, saw the additional PR with the updated text). Perfect - thanks @michaeljenkins88 !

**Pull request merged**